collect_app/src/main/java/org/odk/collect/android/tasks/InstanceUploaderTask.java

```diff
@@ -490,8 +497,16 @@ private boolean uploadOneSubmission(String urlString, String id, String instance
                         outcome.mResults.put(id, fail + response.getStatusLine().getReasonPhrase()
                                 + " (" + responseCode + ") at " + urlString);
                     } else {
-                        outcome.mResults.put(id, fail + response.getStatusLine().getReasonPhrase()
-                                + " (" + responseCode + ") at " + urlString);
+                        // Custom messaging response if fail.
+                        // If Custom Server Response enabled then confirm response is valid
```

Comment:
I removed the the incorrect comments. Thanks. I changed it for the one below aswell.